db/db.go

```diff
@@ -6438,6 +6438,38 @@ func GetAsyncTaskByNameAndBrokerName(task, brokerName string) (bool, []models.As
 	return true, asyncTask, nil
 }
 
+func GetAllAsyncTasks(tenantName string) ([]models.AsyncTaskRes, error) {
+	ctx, cancelfunc := context.WithTimeout(context.Background(), DbOperationTimeout*time.Second)
+	defer cancelfunc()
+
+	conn, err := MetadataDbClient.Client.Acquire(ctx)
+	if err != nil {
+		return []models.AsyncTaskRes{}, err
+	}
+	defer conn.Release()
+
+	query := `SELECT id, name, created_at FROM async_tasks where tenant_name = $1`
```

Comment:
To need to return the tasks ordered by date so that the the newest is first(by created at).